provider/marathon.go

```diff
@@ -20,6 +20,7 @@ type Marathon struct {
 	BaseProvider   `mapstructure:",squash"`
 	Endpoint       string
 	Domain         string
+	DefaultExpose  bool
```

Comment:
I would tend to prefer `ExposedByDefault` than `DefaultExpose` but that's just a nit I think.